patches/ava+5.3.1.patch

```diff
@@ -94,6 +94,23 @@ index 7630baa..78ced77 100644
  
  		notifyOfPeerFailure() {
  			send({type: 'peer-failed'});
+diff --git a/node_modules/ava/lib/reporters/default.js b/node_modules/ava/lib/reporters/default.js
+index 804e285..06ce6c3 100644
+--- a/node_modules/ava/lib/reporters/default.js
++++ b/node_modules/ava/lib/reporters/default.js
+@@ -678,8 +678,10 @@ export default class Reporter {
+ 			this.lineWriter.writeLine(colors.todo(`${this.stats.todoTests} ${plur('test', this.stats.todoTests)} todo`));
+ 		}
+ 
+-		if (this.stats.unhandledRejections > 0) {
+-			this.lineWriter.writeLine(colors.error(`${this.stats.unhandledRejections} unhandled ${plur('rejection', this.stats.unhandledRejections)}`));
++		if (this.stats.unhandledRejections > 0 || process.env.AGORIC_AVA_EXPECT_UNHANDLED_REJECTIONS) {
```

Comment:
Patches make maintenance more costly. In another PR we were able to ditch AVA patching entirely, so this would be a new hurdle.

I was excited to [find a question on StackOverflow about this](https://stackoverflow.com/a/66168704/46040). Have you seen it? ;-)

Did you try the suggestion of running in a child process or worker instead?

Reply:
I am running each test that expects a certain number of unhandled rejections in an AVA child process, but that still requires a patch, or else I don't know how to discriminate between other failures that exit with status code 1.

Reply:
With @gibson042's suggestion, I can use `process.on` event handlers within `ava-unhandled-rejections.js`, so this AVA patch need not live very long.